.gitpod.yml

```diff
@@ -1,15 +1,12 @@
-image: gitpod/workspace-full:2022-08-04-13-40-17
+image: 
+  file: gitpod.Dockerfile
 workspaceLocation: observability/observability.code-workspace
 tasks:
-  - init: |
-      # TODO(arthursens): We should update the workspace image so we dont need the steps below
-      sudo curl -Lo /usr/bin/yq https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64 && sudo chmod +x /usr/bin/yq
-      sudo apt update ; sudo apt-get install shellcheck apt-transport-https ca-certificates -y ; sudo update-ca-certificates
+  - name: Install pre-commit hooks & generate manifests
+    command: |
       make --always-make
```

Comment:
```suggestion
```

Totally unrelated to your PR, but this is the reason our prebuilds are failing.

We use `go get` to install a bunch of tools and we broke that when we updated the image to go 1.19. We can use `go install` instead but those tools were very focused on jsonnet... we don't really need it.

I'll open another PR removing this line